src/layout/header/LoginButton.tsx

```diff
@@ -21,7 +21,7 @@ const LoginButton: FC<LoginButtonProps> = ({ variant }) => {
 
   return (
     <Button color="primary" variant={variant} data-testid="menu-login-button" onClick={handleLogin}>
-      {t('common.login')}
+      {t('common.login').toUpperCase()}
```

Comment:
Hadde det gått an å sette text-transform: uppercase; i Button theme? da slipper vi å passe på å sette uppercase selv i fremtiden?

Reply:
det må jeg sjekke, ja

Reply:
worked...
Skal vi også la personnavn gå til uppercase ?  (jeg synes det ser greit ut)
![image](https://user-images.githubusercontent.com/46489468/123094700-7f57a800-d42d-11eb-964c-8668641e6cef.png)

Reply:
Synes det ser greit ut for min del.

Reply:
Hvis vi får klager så kan vi vel sette stiling på akkurat den knappen til text-transform: none